src/main/java/io/github/sornerol/chess/pubapi/domain/game/PlayerDetail.java

```diff
@@ -36,4 +37,12 @@ public class PlayerDetail {
      */
     @JsonProperty("username")
     private String username;
+    
+    /**
+     * The uuid of the Player
+     */
+    //TODO Check if the String can replace throw a UUID
+    @ApiStatus.Experimental
```

Comment:
I started a discussion on the use of the `@ApiStatus.Experimental` annotation on https://github.com/sornerol/chess-com-pubapi-java-wrapper/pull/13.

Can you point to any documentation from Chess.com to indicate that this is an experimental field? I didn't see anything indicating this when viewing their docs.

Reply:
Please remove the @ApiStatus.Experimental annotations. This is not the intended use of this annotation.